msal/application.py

```diff
@@ -320,6 +358,7 @@ def get_authorization_request_url(
             scope=decorate_scope(scopes, self.client_id),
             nonce=nonce,
             domain_hint=domain_hint,
+            claims=claims,
```

Comment:
(continue from above)
or we can do it inline:

```suggestion
            claims=_merge_claims_and_capabilities(self.client_capabilities, claims),
```